src/main/java/duke/task/ToDo.java

```diff
@@ -0,0 +1,37 @@
+package duke.task;
+
+/**
+ * Class that encapsulates To Do task.
+ *
+ * @author Elgin
+ */
+public class ToDo extends Task {
+    /**
+     * Constructor of the To Do task.
+     *
+     * @param taskName The title of the Task
+     */
+    public ToDo(String taskName) {
+        super(taskName);
+    }
+
+    /**
+     * Constructor of the To Do task.
+     *
+     * @param taskName The title of the task.
+     * @param isDone True if task is already done, false otherwise.
+     */
+    public ToDo(String taskName, boolean isDone) {
+        super(taskName, isDone);
+    }
+
+    /**
+     * String representation of a ToDo.
+     *
+     * @return ToDo string representation.
+     */
```

Comment:
Since you are overriding this method, you can explore using `@inheritDoc` which will help you inherit the JavaDocs from the parent method